scipy/optimize/tests/test_constraints.py

```diff
@@ -76,12 +76,23 @@ def test_prepare_constraint_infeasible_x0():
     bounds = Bounds(lb, ub, enforce_feasibility)
     pytest.raises(ValueError, PreparedConstraint, bounds, x0)
 
+    # check is_satisifed method
```

Comment:
typo in satisfied (and same typo a few more times below)

Reply:
the comments seem to have been removed now, so marking as resolved